libs/Microsoft.MixedReality.WebRTC/PeerConnection.cs

```diff
@@ -1159,10 +1159,31 @@ public bool CreateAnswer()
             return (PeerConnectionInterop.PeerConnection_CreateAnswer(_nativePeerhandle) == Utils.MRS_SUCCESS);
         }
 
+        /// <summary>
+        /// Set the bitrate allocated to all RTP streams sent by this connection.
+        /// Other limitations might affect these limits and are respected (for example
+        /// "b=AS" in SDP).
+        /// </summary>
+        /// <param name="minBitrateBps">Minimum bitrate.</param>
+        /// <param name="startBitrateBps">Start/current target bitrate.</param>
+        /// <param name="maxBitrateBps">Maximum bitrate.</param>
+        public void SetBitrate(uint? minBitrateBps = null, uint? startBitrateBps = null, uint? maxBitrateBps = null)
+        {
+            ThrowIfConnectionNotOpen();
+            int signedMinBitrateBps = minBitrateBps.HasValue ? (int)minBitrateBps.Value : -1;
+            int signedStartBitrateBps = startBitrateBps.HasValue ? (int)startBitrateBps.Value : -1;
+            int signedMaxBitrateBps = maxBitrateBps.HasValue ? (int)maxBitrateBps.Value : -1;
+            if (PeerConnectionInterop.PeerConnection_SetBitrate(_nativePeerhandle,
+                signedMinBitrateBps, signedStartBitrateBps, signedMaxBitrateBps) != Utils.MRS_SUCCESS)
+            {
+                throw new InvalidOperationException();
+            }
```

Comment:
```cs
uint res = PeerConnectionInterop.PeerConnection_SetBitrate(...);
Utils.ThrowOnErrorCode(res);
```
This makes customizing the mapping {error code} <-> {exception} must easier, for example if we add new exception types in the future, and keeps it consistent.